config/environments/production.rb

```diff
@@ -61,12 +61,17 @@
   # require 'syslog/logger'
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
+  if ENV["RAILS_LOG_TO_STDOUT"].present?
+    config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
+  end
```

Comment:
It's not deployed yet on heroku. Though we can remove the Rails12 factor gem when it is